frontend/public/components/container.jsx

```diff
@@ -143,6 +143,8 @@ const Details = (props) => {
           <dd><Timestamp timestamp={state.startedAt} /></dd>
           <dt>Finished</dt>
           <dd><Timestamp timestamp={state.finishedAt} /></dd>
+          <dt>Pod</dt>
+          <dd><ResourceLink kind="Pod" name={props.match.params.podName} title={props.match.params.podName} /></dd>
```

Comment:
I'd remove the title attribute if it's the same as the pod name.

I think we need to specify the namespace, correct? Does the link work without it?

Reply:
It works without it, but it looks like it's checking all namespaces for it. I added the namespace and removed the title in my new commit.